docs/0.4/concepts/splinter_registry.md

```diff
@@ -0,0 +1,227 @@
+using the `--registries` CLI option and the `registries` config file option.
+Each registry is specified with a URL:
+
+* For remote YAML files, the URL should begin with `http://` or `https://`.
+* For local YAML files, the URL should be the path to the file on the local
+  filesystem, prefixed with `file://`.
+
+When using the CLI option, registries can be specified either as a
+comma-separated list, or with multiple uses of the option. These examples are
+equivalent:
+
+``` console
+$ splinterd ... \
+  --registries file:///path/to/local_registry.yaml,https://www.example.com/remote_registry.yaml \
+  ...
+```
+
+``` console
+$ splinterd ... \
+  --registries file:///path/to/local_registry.yaml \
+  --registries https://www.example.com/remote_registry.yaml \
+  ...
+```
+
+When using the config file option, the registries should be specified as a TOML
+array. This example is equivalent to the CLI examples:
+
+``` toml
+registries = ["file:///path/to/local_registry.yaml, https://www.example.com/remote_registry.yaml"]
+```
+
+### Remote YAML File Configuration
+
+The splinter daemon provides two options for configuring the behavior of remote
+YAML file registries:
+
+* The automatic refresh interval is configured with the
+  `--registry-auto-refresh` CLI option and `registry_auto_refresh` config file
+  option. The default value is 600 seconds (10 minutes).
+
+* The forced refresh time is configured with the `--registry-forced-refresh` CLI
+  option and `registry_forced_refresh` config file option. The default value is
+  10 seconds.
+
+Both the automatic refresh interval and the forced refresh time take the number
+of seconds as an argument. These CLI and config file examples are equivalent:
+
+``` console
+$ splinterd ... \
+  --registries https://www.example.com/remote_registry.yaml \
+  --registry-auto-refresh 60 \
+  --registry-forced-refresh 5 \
+  ...
+```
+
+``` toml
+registries = ["https://www.example.com/remote_registry.yaml"]
+registry_auto_refresh = 60
+registry_forced_refresh = 5
+```
+
+For more information on the automatic and forced refreshes, see
+[Types of Source Registries](#types-of-source-registries) above.
+
+## Accessing and Updating the Registry
+
+The splinter daemon's registry can be accessed via the REST API. The following
+registry endpoints are provided by splinterd:
+
+* `POST /registry/nodes` adds a node to the registry
+* `GET /registry/nodes` lists nodes in the registry
+* `GET /registry/nodes/{identity}` fetches a node in the registry by its
+  identity
+* `PUT /registry/nodes/{identity}` adds or replaces a node in the registry
+* `DELETE /registry/nodes/{identity}` deletes a node from the registry
+
+For more information, see the
+<a href="/docs/0.4/api/#tag/Splinter-Registry" target="_blank">
+Splinter registry routes REST API reference
+</a>.
+
+## Future considerations
+
+### Database Registries
+
+Splinter currently provides three registry implementations:
+
+* Local YAML file
+* Remote YAML file
+* Unified
```

Comment:
This feels like it needs a noun after the adjective.  How about:

- Unified registry with changes cached in the local YAML file

Reply:
This section has been removed